templates/main.php

```diff
@@ -165,6 +165,36 @@
 					endwhile;
 				}
 			}
+		} else { // if BuddyPress.
+			?>
+			<div id="item-body">
+			<?php
+		}
+
+		rtmedia_load_template();
+
+		if ( ! $rt_ajax_request ) {
+			if ( function_exists( 'bp_displayed_user_id' ) && 'buddypress' === $template_type && ( bp_displayed_user_id() || bp_is_group() ) ) {
```

Comment:
I think here no need to put condition for this two functions: `bp_displayed_user_id() || bp_is_group()` as they are already getting checked in below if condition.